lib/cors/validate.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+'use strict'
+
+const validUrl = require('valid-url')
+
+/* ::
+import type {CorsConfiguration} from './read.js'
+*/
+
+const HELP = ', see documentation for information on how to configure cors.'
+
+function validateCors (
+  cors /* : CorsConfiguration */
+) /* : Promise<CorsConfiguration> */ {
+  if (!cors) {
+    return Promise.reject(new Error('Must specify cors configuration' + HELP))
```

Comment:
It should be okay to not have a CORS configuration. After all, that's what machine-to-machine or Lambda-inception use cases, or even just the common case of native-only front-ends will be likely to do.